templates/article.liquid

```diff
@@ -1,18 +1,87 @@
-<h2>{{ article.title }}</h2>
-<p>{{ article.published_at | date: '%d %B %Y' }}</p>
-{% if blog.handle == 'lookbook' %}
-
-  {% if article.content contains "<img" %}
-    {% assign gallery-src = article.content | split: 'src="' %}
-    {% assign gallery-size = gallery-src.size | minus:1 %}
-    {% for i in (1..gallery-size) %}
-      <h3>This is a lookbook</h3>
-      <li class="lookbook-image">
-        <img src="{{gallery-src[i]}}" />
-      </li>
-    {% endfor %}
-  {% endif %}
-
-{% else %}
-  {{ article.content }}
-{% endif %}
+{% assign dash = '&nbsp;&nbsp;&mdash;&nbsp;&nbsp;' %}
+
+<article class="container">
+  <div class="grid-row justify-center">
+    <div class="grid-item item-s-12  item-m-7">
+      <div class="single-post-details margin-bottom-small">
+        <span class="date">{{ article.published_at | date: '%b %e, %Y' }}</span>
+        <h1 class="article-title margin-bottom-small">
+          {{ article.title }}
+        </h1>
+        <p>
```

Comment:
This excerpt should have a 1 (maybe 2) column indent in relation to the tags below

Reply:
This will be added whenever this gets styled

Reply:
this is scaffold tho as relies on the grid